### ISSUE TITLE
Bundling app with Webpack, cannot resolve 'fs'

### ISSUE THREAD

**Issue by tomoakley** [richorama/country-code-lookup#8]
When bundling my app with webpack, and your library is being used, I get the following error:

```
Module not found: Error: Can't resolve 'fs' in '/path/to/project/node_modules/country-code-lookup'
@ ./node_modules/country-code-lookup/index.js 6:12-25
```

I have tried to use the suggestions listed [here](https://github.com/webpack-contrib/css-loader/issues/447) but they don't work and mostly just come up with the various other errors reported (e.g `Can't resolve variable require`, etc).

Do you have suggestions on how to fix this?

**Comment by richorama** [comment]
Thanks for reporting. I think I could probably rewrite the code to avoid `require`ing anything.

**Comment by richorama** [comment]
@tomoakley this should be fixed in version 0.0.9

**Comment by tomoakley** [comment]
Wow, thanks for the super quick response and fix! This works perfectly.
